play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -132,8 +132,18 @@ basic computer inquiry or operation. +10% to ONE pertinent class skill
 (hacking, etc.) while using this feat, to be chosen at the time of taking 
 this feat.
 
-First Shot Counts: 
+Fish To Water
 Prerequisites:
+Maybe you took a lot of swimming lessons when you were little, maybe you
+were on the college swim team, or maybe your military training had an 
+emphasis on endurance and core strength developed in the pool. In any case,
+you gain 10 skill in swimming and a bonus +15 to checks when holding your
+breath. You also ignore movement penalties from wading through deep liquids.
+These bonuses increase in an additional 10 skill in swimming and an additional
++5 bonus to holding one's breath after level 7.

```

Comment:
Can you change the wording to "These bonuses increase to 20 skill in swimming and a +20 bonus to holding one's breath after level 7."

Keeping the player from having to do math will be beneficial :)
I really like this leveling feature!

Reply:
I really like that wording better. Will change! :)